bdk_core/src/lib.rs

```diff
@@ -24,7 +24,7 @@ extern crate std;
 extern crate hashbrown;
 
 /// Block height and timestamp of a block
-#[derive(Debug, Clone, PartialEq, Eq, Default, Copy)]
+#[derive(Debug, Clone, PartialEq, Eq, Default, Copy, PartialOrd, Ord)]
```

Comment:
I did this because of a compilation error somewhere.. I didn't look into details but probably due to how I was using blocktime in some mapings. This was the quick easy fix so I went with that..

But isn't the correct ordering of `BlockTime` is just increasing heights??